test/core/origin_core/commit_metablock.js

```diff
@@ -0,0 +1,527 @@
+// Copyright 2018 OpenST Ltd.
```

Comment:
Name of this file should be `commitMetaBlock.js`; in line with current practice. We could change all test file names to snake-case, though...

Reply:
We should discuss this 👍

Reply:
Make all tests snake case please